gerrymandering/gerrymandering.ipynb

```diff
@@ -80,6 +84,7 @@
     "color_key = {'w':'blue',  'b':'green', 'a':'red',   'h':'orange',   'o':'saddlebrown'}\n",
     "races     = {'w':'White', 'b':'Black', 'a':'Asian', 'h':'Hispanic', 'o':'Other'}\n",
     "\n",
+    "# Should we get a legend for free when setting datashade=True?\n",
```

Comment:
No, the legend should come for free when setting `rasterize=True`; we don't have any plans for supporting legends automatically when `datashade=True` (since Datashader is in charge of the legend then, not HoloViews or Bokeh). But note that the automatic legend will show e.g. "w" instead of "white", because that's what's in the actual dataframe's column. @jlstevens might recall whether that's been addressed, e.g. by accepting something like `dict(w="White", ...)`